docs/userguide/execution.rst

```diff
@@ -47,8 +47,7 @@ Parsl currently supports the following providers:
 7. `parsl.providers.AWSProvider`: This provider allows you to provision and manage cloud nodes from Amazon Web Services.
 8. `parsl.providers.GoogleCloudProvider`: This provider allows you to provision and manage cloud nodes from Google Cloud.
 9. `parsl.providers.KubernetesProvider`: This provider allows you to provision and manage containers on a Kubernetes cluster.
-10. `parsl.providers.AdHocProvider`: This provider allows you manage execution over a collection of nodes to form an ad-hoc cluster.
-11. `parsl.providers.LSFProvider`: This provider allows you to schedule resources via IBM's LSF scheduler.
```

Comment:
I had thought that both Markdown and reStructured text followed the utility of just enumerating with the number 1:
```reST
1. list item 1
1. list item 2
1. list item 3
...
1. list item N
```
Both monotonically increasing and single numbers work, but for just instances like this, it's a shame the original author of this section didn't just use `1.`

:shrug: